kuma/api/v1/urls.py

```diff
@@ -9,10 +9,10 @@
     re_path(r"^search/(?P<locale>[^/]+)/?$", views.search, name="api.v1.search"),
     re_path(r"^bc-signal/?$", views.bc_signal, name="api.v1.bc_signal"),
     re_path(r"^users/(?P<username>[^/]+)/?$", views.get_user, name="api.v1.get_user"),
-    re_path(
-        r"^subscriptions/feedback/$",
+    path(
+        "subscriptions/feedback/",
         views.send_subscriptions_feedback,
         name="api.v1.send_subscriptions_feedback",
     ),
-    re_path(r"^subscriptions/$", views.subscriptions, name="api.v1.subscriptions"),
+    path("subscriptions/", views.subscriptions, name="api.v1.subscriptions"),
```

Comment:
So much more convenient. We should be using `path()` instead of `re_path()` wherever we can instead. 
The reason we're not already using it everywhere is because we didn't have the option when we upgraded to Django 2. Had we attempted to upgrade to Django 2 and fix all the `urlpatterns` we'd be taking on too much of a risk.